.github/workflows/release.yml

```
@@ -15,13 +15,16 @@ concurrency:
   cancel-in-progress: false  # Never cancel an in-flight release
 
 permissions:
-  contents: write
-  id-token: write
+  contents: read
+  id-token: read
```

actionlint (1.7.11)

[error] "read" is invalid as permission of scope "id-token". available values are "write", "none"

(permissions)